src/rpc/rawtransaction.cpp

```diff
@@ -760,7 +760,9 @@ static UniValue signrawtransactionwithkey(const JSONRPCRequest& request)
 static UniValue sendrawtransaction(const JSONRPCRequest& request)
 {
     RPCHelpMan{"sendrawtransaction",
-                "\nSubmits raw transaction (serialized, hex-encoded) to local node and network.\n"
+                "\nSubmit a raw transaction (serialized, hex-encoded) to local node and network.\n"
+                "Note that the transaction will be sent unconditionally to all peers, so using\n"
+                "this for manual rebroadcast may represent a privacy leak.\n"
```

Comment:
maybe add reason why "as nodes will not normally ever re-broadcast transactions already in their mempool"

Reply:
Though, this is about to change soon. (hopefully)

Reply:
Thank you for reviewing; good idea! Added (after checking spelling of re-broadcast vs rebroadcast, wasn't sure myself), updated commit message and PR description.

Reply:
Also, not true for wallet txs today

Reply:
"as nodes will normally not rebroadcast (non-wallet) transactions already in their mempool." ?